specification/resources/resource-manager/Microsoft.Resources/stable/2016-06-01/subscriptions.json

```diff
@@ -312,6 +337,49 @@
         "nextLink"
       ],
       "description": "Tenant Ids information."
+    },
+    "Operation": {
+      "description": "Microsoft.Resources operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "description": "The object that represents the operation.",
```

Comment:
Do you mean the description here? I already had it for display.

Reply:
"description" the Swagger keyword, and "description" the JSON key in ARM answer are too different things. You have a Swagger "description" of "display", but "display" can contain a property called "description" which has to be defined:
```json
  "display": {
    "provider": "{Name of the provider for display purposes}",
    "resource": "{Name of the resource type for display purposes}",
    "operation": "{Name of the operation for display purposes}",
    "description": "{Description of the operation for display purposes}"
  },
```

Example:
https://github.com/Azure/azure-rest-api-specs/blob/57d827ec31e84a77fa35dfe6bdd048eb84916918/specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/storage.json#L581-L601

Reply:
Thanks, it makes sense to me :)